gecoscc/api/help_channel_client.py

```diff
@@ -0,0 +1,374 @@
+#
+# Copyright 2018, Junta de Andalucia
+# http://www.juntadeandalucia.es/
+#
+# Authors:
+#   Abraham Macías <amacias@gruposolutia.com>
+#
+# All rights reserved - EUPL License V 1.1
+# https://joinup.ec.europa.eu/software/page/eupl/licence-eupl
+#
+
+import urllib2
+import datetime
+import os
+import string
+import re
+import socket
+from random import randint, choice
+from cornice.resource import resource
+from gecoscc.utils import get_chef_api
+from chef import ChefError
+from chef.exceptions import ChefServerError
+from chef import Client as ChefClient
+from chef import Node as ChefNode
+from Crypto.PublicKey import RSA
+from bson import ObjectId
+
+import logging
+logger = logging.getLogger(__name__)
+
+known_message = 'En un lugar de la Mancha, de cuyo nombre no quiero acordarme,'\
```

Comment:
?

Reply:
I will accept this for testing and think about setting that known secret in gecoscc.ini later